src/main/java/thePackmaster/actions/graveyardpack/ExhaustFromHandAction.java

```diff
@@ -0,0 +1,151 @@
+package thePackmaster.actions.graveyardpack;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.common.DrawCardAction;
+import com.megacrit.cardcrawl.actions.common.GainEnergyAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.core.Settings;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.localization.UIStrings;
+
+public class ExhaustFromHandAction extends AbstractGameAction {
```

Comment:
This action is a bit of a mess. If I understand it correctly, you just want to exhaust cards from hand without a screen.

This might be more useful for a full character but since it's only used once in this character, I'd rather not deal with it. All that card does is exhaust all cards in hand, so please use an anonymous action `new AbstractGameAction() {...` call ExhaustSpecificCard action for every card in hand which is how base game does it.